ClientApp/src/components/EnergyCalculation.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+
+export default function EnergyCalculation() {
+  const [cost, getCost] = useState([]);
```

Comment:
Following the code standard used in previous commits this should be ``[cost, setCost]``

Reply:
Changed in new commit